app/src/main/java/com/example/kotlin/robertoruizapp/framework/view/activities/MainActivity.kt

```diff
@@ -0,0 +1,87 @@
+package com.example.kotlin.robertoruizapp.framework.view.activities
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import com.example.kotlin.mypokedexapp.viewmodel.MainViewModel
+import com.example.kotlin.robertoruizapp.R
+import com.example.kotlin.robertoruizapp.databinding.ActivityMainBinding
+import com.example.kotlin.robertoruizapp.framework.view.fragments.FragmentoCursos
+import com.example.kotlin.robertoruizapp.framework.view.fragments.FragmentoHome
+import com.example.kotlin.robertoruizapp.utils.Constants
+
+class MainActivity: AppCompatActivity() {
+
```

Comment:
Quitar espacios en blanco

Reply:
esto se puede hacer automáticamente, recuerden utilizar `ctrl + alt + shift + L`
Les recomiendo mucho que lean esto, es cortito y muy útil
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat_file